www/html/ci/application/views/landing.php

```diff
@@ -578,7 +578,7 @@ function googleTranslateElementInit() {
 <p>Do you want to play an arcade game that has you flying a jetplane while playing epic music in space? Avoid the space obstacles and enemies and get to the end of the song! If you get hit, the space gods will give you another chance by having you press a keyboard key within a certain time limit. If you press it in time, then you are safe! If not, then I guess you are starting the song over again. Inspired by my childhood space arcade games and The Impossible Game.</p>
 
 <p><strong>Watch the trailer </strong><a href="https://youtu.be/tYUymiCDOY8" target="_blank">here!</a></p>
-<p><strong>Download the game! </strong><a href="http://www.mediafire.com/file/7fdkazh3w61l6dc/SpaceDodgersGithub.zip" target="_blank">WindowsOS</a></p>
+<p><strong>Download the game! </strong><a href="downloads/SpaceDodgersGithub.zip" download="SpaceDodgers">WindowsOS</a></p>
```

Comment:
@finn00dev Is the name of the downloaded file "SpaceDodgers" or "SpaceDodgersGithub.zip"? Let's not change the downloaded filename.